ducktape/cluster/remoteaccount.py

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import paramiko
+# Constant that is responsible for updating ssh keys after
```

Comment:
nit: SSH key means something else in this context (public key used for authentication) ..think you mean session key? (which is used to encrypt the data in that connection), may be clarify that?